script.js

```diff
@@ -0,0 +1,71 @@
+let defaultResult = 0;
+
+let currentResult = defaultResult;
+
+let logEntries = [];
```

Comment:
u can use const here ?

Reply:
is this related to primitive type vs reference type ? I haven't studied it yet. will see in future. @anask601